package.json

```diff
@@ -23,15 +28,13 @@
   },
   "main": "index.js",
   "devDependencies": {
-    "chai": "^2.0.0",
+    "chai": "^3.5.0",
     "mocha": "^2.1.0",
     "mocha-fibers": "^1.1.0",
-    "ws": "^0.7.1"
+    "ws": "^1.1.0"
   },
   "directories": {
-    "doc": "docs",
-    "example": "examples",
-    "test": "test"
+    "doc": "docs"
```

Comment:
This is the only valid field of the 3 that you had, per https://docs.npmjs.com/files/package.json#directories